app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/media/VideoMediaPlayerGlue.java

```diff
@@ -254,6 +383,37 @@ public void releaseResources() {
         }
     }
 
+    public void createMediaSessionIfNeeded() {
+        if (mVideoSession == null) {
+            mVideoSession = new MediaSession(this.getContext(), "VideoPlayer Session");
```

Comment:
So we're creating a new MediaSession. Is this the right thing to do if we're recreating the activity due to a configuration change, or resuming playback from a now playing entry if the app was nuked in the background?